app/controllers/users/approvals_controller.rb

```diff
@@ -22,4 +26,15 @@ def reject
     render "users/approvals/approve"
   end
 
+  def check_user
```

Comment:
@t-harps Ah ok I see what you mean
`index` uses `current_user` to retrieve the list of approvals. Therefore, you can't get someone else's info.
However, there's obviously the need to check that the developer is sending their user to the correct URL. So maybe have the method deal with that more specifically?